packages/components/src/FormField/FormField.css

```diff
@@ -69,9 +69,14 @@
   --field--padding-bottom: var(--space-small);
 }
 
+.miniLabel.large {
+  --field--padding-top: calc(var(--space-large) + var(--space-smaller));
+}
+
 .wrapper.textarea {
   height: auto;
   min-height: auto;
+  --field--placeholder-offset: 0;
```

Comment:
actually, when would this be not 0?

Reply:
There are scenarios with regular text inputs that this was doing stuff for from my testing.

Reply:
Hmmm... do you know what those are? Looking at it from the UI perspective, it seems that it always uses a top of 0? 🤔 

Just asking because to Matt's point, it should be the same styling regardless of textarea or input.

Reply:
@darryltec let me get double check and I can add screenshots